tactics/tacticals.ml

```diff
@@ -366,6 +366,16 @@ module New = struct
           catch_failerror e <*> t2
         end
     end
+
+  let tclORELSE0L t1 t2 =
```

Comment:
It's small enough that I wouldn't care about duplication.

It may be worth is, some day, to benchmarki to see if we lose any performance by interpreting `tclINDEPENDENT` in terms of `tclINDEPENDENTL` and `tclORELSE0` in terms of `tclORELSE0L`. But that's not urgent.